docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -1730,10 +1636,7 @@ _Returns_
 
 ### validateBlocksToTemplate
 
-Block validity is a function of blocks state (at the point of a
-reset) and the template setting. As a compromise to its placement
-across distinct parts of state, it is implemented here as a side-
-effect of the block reset action.
+Block validity is a function of blocks state (at the point of a reset) and the template setting. As a compromise to its placement across distinct parts of state, it is implemented here as a sideeffect of the block reset action.
```

Comment:
This is where the hyphenation is removed. Perhaps it should be replaced with a single-byte space.

```
// From
side-
effect

// To
sideeffect
```

Reply:
yes, as a 30+ year native english speaker, it should be "side effect"

Reply:
Thanks for the advice. As you said, I don't think this is the hyphenation, so I fixed the JSDoc comment and regenerated the document in [8ba9724](https://github.com/WordPress/gutenberg/pull/49635/commits/8ba972462b66ca1bc715e5190c133ac6113ab926).